docs/_ext/scylladb_cc_properties.py

```diff
@@ -161,12 +166,36 @@ def readable_desc(description: str) -> str:
         .replace('"', "")
     )
 
+def readable_desc_rst(description):
+    indent = ' ' * 3
+    lines = description.split('\n')
+    cleaned_lines = []
+    
+    for line in lines:
+
+        cleaned_line = line.replace('\\n', '\n')
+
+        if line.endswith('"'):
+            cleaned_line = cleaned_line[:-1] + ' '
+
+        cleaned_line = cleaned_line.lstrip()
+        cleaned_line = cleaned_line.replace('"', '')
+        
+        if cleaned_line != '':
+            cleaned_line = indent + cleaned_line
+            cleaned_lines.append(cleaned_line)
+    
+    cleaned_description = ''.join(cleaned_lines)
```

Comment:
yeah, i think we could use `textwrap` or use jinja's builtin filter for indenting text.. but if it is not used, could drop it from the top of this file.

Reply:
and could just return `''.join(cleaned_lines)` directly without the help of this variable.